migrating_legacy_token_opt_in/contracts/old_erc20_token/MyToken_V0.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.11;
+
+import "./zeppelin-solidity-1.1.0/math/SafeMath.sol";
+import "./zeppelin-solidity-1.1.0/token/PausableToken.sol";
+import "./zeppelin-solidity-1.1.0/token/MintableToken.sol";
+import "./zeppelin-solidity-1.1.0/token/TokenTimelock.sol";
+
+/**
+ * @title MyToken V0
+ */
+contract MyToken_V0 is PausableToken, MintableToken {
+  using SafeMath for uint256;
+
+  string public name = "MyToken";
+  string public symbol = "MTK";
+  uint public decimals = 18;
+
+  function mintTimelocked(address _to, uint256 _amount, uint256 _releaseTime) onlyOwner canMint returns (TokenTimelock) {
```

Comment:
Why is this function here?

Reply:
I was just mimicking  a real sample object